embedded-nal-async/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "embedded-nal-async"
+version = "0.1.0" # remember to update html_root_url
+authors = [
+    "Jonathan 'theJPster' Pallant <github@thejpster.org.uk>",
+    "Mathias Koch <mk@blackbird.online>",
+    "Diego Barrios Romero <eldruin@gmail.com>",
+    "Ryan Summers <ryan.summers@vertigo-designs.com>",
+    "Ulf Lilleengen <lulf@redhat.com>",
```

Comment:
I don't know if I'd list all of us as the authors of the async NAL at this point - I certainly haven't had a hand in it. So far it's just your work.